api/v1alpha1/perses_types.go

```diff
@@ -22,10 +22,50 @@ import (
 
 // PersesSpec defines the desired state of Perses
 type PersesSpec struct {
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// Client perses client configuration
+	Client Client `json:"client,omitempty"`
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	Config PersesConfig `json:"config,omitempty"`
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	ContainerPort int32 `json:"containerPort,omitempty"`
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// Args extra arguments to pass to perses
+	Args []string `json:"args,omitempty"`
+}
+
+type Client struct {
+	// +optional
+	// TLS tls_config for perses client
+	TLS *TLS `json:"tls,omitempty"`
+}
+
+type TLS struct {
```

Comment:
what is the reason of not using the TLSconfig defined in Perses ? https://github.com/perses/perses/blob/main/pkg/model/api/v1/secret/tls_config.go#L70

Reply:
ah yeah ok got it, it's because of the Certificate struct.

Reply:
@Nexucis It could make sense to rename the field and struct TLSConfig just for a similar naming convention though.